server/chat-commands.js

```diff
@@ -622,19 +622,10 @@ const commands = {
 	idle: 'away',
 	afk: 'away',
 	away(target, room, user) {
-		this.parse('/blockchallenges');
-		this.parse(`/blockpms ${target}`);
+		user.setAway();
+		this.sendReply("You are now marked as away.");
 	},
-	awayhelp: [`/away - Blocks challenges and private messages. Unblock them with /back.`],
-
-	'!back': true,
```

Comment:
Revert? I think we still want this (*I* still want this)?

Reply:
I mean, if you send a message you get marked as back

Reply:
This command would literally just be a placebo

Reply:
but I guess it makes sense to have a command so you don't have to broadcast something to everyone?

Reply:
Yeah, but I want to be able to change state without sending a message. I want to be able to silently indicate that I'm no longer away.